src/types/GTypes/items/index.ts

```diff
@@ -161,6 +162,8 @@ export type ItemKey =
 
 type GItemDynamic = {
   buy: boolean | undefined;
+  upgrade: Partial<Record<StatType, number>>;
+  compound: Partial<Record<StatType, number>>;
```

Comment:
Don't we already have a type for `Partial<Record<StatType, number>>`? I think it should probably be a type in the same file as `StatType`.

Reply:
I actually don't know, not sure if it belongs in stat type though, it's something item specific.

I added it here instead of adding it to all the different items.